modules/vaos/app/serializers/ccra/referral_list_serializer.rb

```diff
@@ -7,11 +7,12 @@ class ReferralListSerializer
     include JSONAPI::Serializer
 
     set_key_transform :camel_lower
-    set_id :uuid
+    set_id :referral_consult_id
```

Comment:
This also needs to stay as uuid

Reply:
that can't be uuid, the value has to be the referral_consult_id